src/filter/exceptions.filter.ts

```diff
@@ -0,0 +1,27 @@
+import { ExceptionFilter, Catch, ArgumentsHost, HttpException, UnauthorizedException, BadRequestException } from '@nestjs/common';
+import { Response } from 'express';
+import { ErrorResponse } from '../dto/error-response.dto';
+
+@Catch()
+export class ExceptionsFilter implements ExceptionFilter {
```

Comment:
In case of Internal Server Error, we do not return a message information. We will be returning the error to the user automatically in this case. A fixed error message can be written instead. Let's meet @mfozmen